src/TesApi.Web/ConfigurationUtils.cs

```diff
@@ -82,20 +87,20 @@ public async Task<List<string>> ProcessAllowedVmSizesConfigurationFileAsync(Canc
             }
             catch
             {
-                logger.LogWarning($"Failed to write {supportedVmSizesUrl.AbsolutePath}. Updated VM size information will not be available in the configuration directory. This will not impact the workflow execution.");
+                logger.LogWarning("Failed to write {SupportedVmSizesUrl}. Updated VM size information will not be available in the configuration directory. This will not impact the workflow execution.", supportedVmSizesUrl.AbsolutePath);
             }
 
             var allowedVmSizesFileContent = await storageAccessProvider.DownloadBlobAsync(allowedVmSizesUrl, cancellationToken);
 
             if (allowedVmSizesFileContent is null)
             {
-                logger.LogWarning($"Unable to read from {allowedVmSizesUrl.AbsolutePath}. All supported VM sizes will be eligible for Azure Batch task scheduling.");
+                logger.LogWarning("Unable to read from {AllowedVmSizesUrl}. All supported VM sizes will be eligible for Azure Batch task scheduling.", allowedVmSizesUrl.AbsolutePath);
                 return supportedVmSizes.Select(v => v.VmSize).Distinct().ToList();
             }
 
             // Read the allowed-vm-sizes configuration file and remove any previous warnings (those start with "<" following the VM size or family name)
             var allowedVmSizesLines = allowedVmSizesFileContent
-                .Split(new[] { '\r', '\n' })
+                .Split(['\r', '\n'])
```

Comment:
Going forward, let's agree not to make changes like this unless that line of code is being changed for other reasons